From: Aaron Seigo <aseigo@kde.org>
Date: Mon, 08 Jul 2013 10:29:35 +0000
Subject: don't build the dirmodel export if building against a new enough KDE SC
X-Git-Url: http://quickgit.kde.org/?p=plasma-mobile.git&a=commitdiff&h=07ab893ab13a0bdf03c57c5000d5aa1ff82716ef
---
don't build the dirmodel export if building against a new enough KDE SC

in 4.11 and above the dirmodel plugin was moved to kde-runtime

BUG:322063
---


--- a/components/CMakeLists.txt
+++ b/components/CMakeLists.txt
@@ -1,4 +1,11 @@
 add_subdirectory(metadatamodel)
 add_subdirectory(mobilecomponents)
-add_subdirectory(dirmodel)
 add_subdirectory(settings)
+ 
+# dirmodel can be removed once plasma-mobile hard-requires KDE libs/runtime 4.11
+# as long as 4.10 is the requirement, however, we need to keep a copy of it here
+macro_ensure_version("4.10.80" ${KDE_VERSION} HAS_DIRMODEL_IN_RUNTIME)
+if (!${HAS_DIRMODEL_IN_RUNTIME})
+    add_subdirectory(dirmodel)
+endif (!${HAS_DIRMODEL_IN_RUNTIME})
+

